src/usr/local/www/classes/Form/Checkbox.class.php

```diff
@@ -56,9 +56,9 @@ protected function _getInput()
 	{
 		$input = parent::_getInput();
 
-		if (empty($this->_description))
-			return '<label class="chkboxlbl">'. $input .'</label>';
+		if (!empty($this->_description) || is_numeric($this->_description))
```

Comment:
Maybe we should patch the pfSense FreeBSD PHP port so that empty() behaves differently/better :)
To my mind "empty" should mean a zero-length string or the special value "null". Everything else is "something" - text, a number (even if it is zero), a boolean value true or false...
Or maybe a new function nothing() that does what empty() should do.